test/e2e/memcached_test.go

```diff
@@ -316,11 +317,23 @@ func memcachedLeaderTest(t *testing.T, f *framework.Framework, ctx *framework.Te
 
 func verifyLeader(t *testing.T, namespace string, f *framework.Framework) (*v1.Pod, error) {
 	// get configmap, which is the lock
+	lockName := "memcached-operator-lock"
 	lock := v1.ConfigMap{}
-	err := f.Client.Get(context.TODO(), types.NamespacedName{Name: "memcached-operator-lock", Namespace: namespace}, &lock)
+	err := wait.Poll(time.Second*5, time.Second*60, func() (done bool, err error) {
```

Comment:
The wait times are they based on anything or just approximate times?

Reply:
They're arbitrary. Seems like a reasonable balance between spamming the API server and delaying our test too much if the leader lock isn't present on the first try.